tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -2755,6 +2741,20 @@ error: Cannot call an optional function
    | ^^^^^^^^^^^^^^^^ Cannot call an optional function
 
 
+error: Expected type to be \\"(preflight (num): void)?\\", but got \\"preflight (x: str): void\\" instead
+    --> ../../../examples/tests/invalid/optionals.test.w:102:53
+    |
+102 | let functionWithOptionalFuncParam1: ((num):void)? = (x: str):void => {};
+    |                                                     ^^^^^^^^^^^^^^^^^^^ Expected type to be \\"(preflight (num): void)?\\", but got \\"preflight (x: str): void\\" instead
```

Comment:
Unrelated to the PR changes but it feels a bit weird that some of the type signatures are formatted with parameter names, and some of the type signatures are formatted without type names

```
preflight (num): void
vs
preflight (x: num): void
```

Reply:
I also noticed this, I'll add a new issue